doc/memos/rdm-draft-petry-riot-design-goals.md

```diff
@@ -0,0 +1,218 @@
+
+ - RDM: 1
```

Comment:
as per RDM0 this should be changed to "rdm-draft-petry-riot-design-goals.md" for now ;)

Reply:
done

Reply:
changed back